docs/individual_sheets.md

```diff
@@ -220,47 +223,53 @@ options for each row are clickable links in the chart below:
     </td>
     <td class="row-4">
       You can put either: 
-      <em style="font-weight: bold">TRUE</em>, <em style="font-weight: bold">FALSE</em>, or <em>you can leave blank for False</em>
+      <em>project name</em> or <em style="font-weight: bold">FALSE</em>
     </td>
     <td class="row-4">
       <em>Your notes</em>
     </td>
   </tr>
 </table>
 
-??? note "use_existing"
-    `use_existing` column field is best for when you already have a material within your project and 
-    you just want to name it and use it within your Excel file instead of creating a new material on every upload 
-
+<details>
+<summary> Notes on use_existing column  </summary>
+The use_existing column field can be used to obtain materials from your own projects or other publicly available projects. If obtaining legacy materials or materials from outside your group, you may find some characteristics of the material to be missing.
+</details>
+<details>
+<summary> Notes on material attributes </summary>
+All attributes material can be added by marking the column as attribute in row 1 and using the attribute name from the <a href=https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/6322994103e27d9176d5b10c/original/main-supporting-information.pdf>data model</a>. 
+Additionally, for the more complex formatting of the computational_forcefield attribute reference the picture below:
+<img src="../docs_assets/comp_forcefield_setup.png" alt="picture of computational forcefield setup">
```

Comment:
how come we decided not to go with HTML table? I think the image is fine, but just wondering